helper-commands.md

```diff
@@ -0,0 +1,91 @@
+> 6. None of the above
+
+
+### `!show`
+> Show your work, and if possible, explain where you are stuck.
+
+### `!onechannel`
+> Please stick to your channel.
+
+### `!volunteers`
+> Helpers are just people volunteering their time to help you. Be polite and patient.
+
+### `!nosols`
+> As a helper, please do not give out answers that could be copied as a homework solution. Have the student work through the problem themselves and guide them along the way.
+
+### `!noadvert`
+> Please do not advertise your help channel or thread in other parts of the server. There are many people who need help, and no one person can be prioritized over other people, so please patiently wait. Anyone who chooses to help you is a volunteer who is doing so out of their own kindness.
+
+### `!noclopen`
+> Please don't repeatedly close and claim a new channel with the exact same question. This erases all previous progress made towards your problem and is confusing for helpers, making it more difficult to help you. Please be patient, even if your channel has not received much attention.
+
+### `!occupied`
+> Someone else is already using this help channel. If you need help with a question, please open your own help channel/thread (see ⁠#how-to-get-help for instructions).
+
+### `!redir`
+> This channel is only for on-topic discussion. Please take casual conversation to ⁠#discussion or ⁠#chill.
+
+### `!patience`
+> Please wait patiently, and do not interrupt other channels with your question. Helpers in this server are volunteers, and the server does not guarantee that someone else will help you. **By being impatient or begging, you will only turn potential helpers away.**
+
+> In the meantime, please make sure your channel contains the original question and clearly describes what you have already tried and what you are having trouble with. This increases your chances of getting a response.
+
+### `!msgdel`
+> The original post of this help channel has been deleted, and it will abruptly close and possibly lock. **(This is irreversible.)** Please claim a new channel, and don't delete the first message of any future channel you claim.
+
+### `!original`
+> Please show the original problem, exactly as it was stated to you. A picture or screenshot is best.
+
+> **If the original problem is not in English, then post it anyway!** The additional context might still help helpers help you. Do your best to translate.
+
+### `!da2a`
+> No need to ask "Can I ask...?" or "Does anyone know about...?"—it's faster for everyone if you just ask your question! See https://dontasktoask.com/
+
+### `!noping`
+> Please do not ping individual helpers unprompted.
+
+### `!nogpt`
+> Please do not trust ChatGPT or similar AI tools for mathematical tasks, as they often generate output which "sounds correct" but has numerous factual or logical errors. Use of these AI tools to answer other people's help questions is strictly against server rules (see #rules).
```

Comment:
Channel link? Also potentially link to `/rules`

Reply:
Added `#rules` mention - this links to `/rules` rather than the channel